config/routes.rb

```diff
@@ -7,12 +7,11 @@
 
   delete '/logout', to: 'sessions#destroy', as: 'logout'
 
-  get '/cart', to: "carts#showcart", as: "showcart"
-
-
   resources :categories
 
-  resources :carts
+  post '/carts', to: 'carts#create', as: 'carts'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.